lua/lvim/utils/functions.lua

```diff
@@ -0,0 +1,17 @@
+local M = {}
+
+M.smart_quit = function()
```

Comment:
```suggestion
M.function smart_quit()
```
nitpick :')

Reply:
are you sure that is the correct syntax?
<img width="847" alt="Screen Shot 2022-05-03 at 6 33 38 PM" src="https://user-images.githubusercontent.com/10992695/166468075-a4d7feba-3441-4753-a6b4-0bf58c59e08a.png">

Reply:
maybe you meant ?
```lua
function M.smart_quit()
```